CONTRIBUTING.md

```diff
@@ -18,7 +18,12 @@ reported the issue. When possible use an existing template and provide all the i
 
 Contributions via pull requests are much appreciated. Before sending us a pull request, please ensure that:
 
-1. You are working against the latest source on the *main* branch.
+1. Make sure you have the following dependency installed:
+- Node.js version 12 or higher. Node.js 16 (Active LTS) is recommended.
+- Typescript version 4.2.3 because later versions may cause reference errors in `lib.d.ts`.
```

Comment:
nit: Typescript -> TypeScript (Please do not throw an hammer at me 😅 ) Also, would it still be helpful for builders to have the TypeScript version in README. It goes to NPM but its not only contributing information but development info for builders app right?